src/diagram/components/quantity-node.scss

```diff
@@ -1,167 +1,358 @@
-@use "../../scss/vars.scss";
-
-$node-width: 180px;
-$node-height: 120px;
-$node-inner-width: 160px;
-$input-half-width: calc(($node-inner-width/2) - 8px);
-$color8:   #e98c42;
-$color8-1: #fefbf9;
-$color8-2: #f9e2d0;
-$color8-3: #f4c5a0;
-$color8-4: #fccb84;
-$color8-5: #da5d1d;
-$color8-hover: $color8-4;
-$color8-selected: $color8-5;
-$gray-mid: #bcbcbc;
+@import "../../scss/vars.scss";
+@import "../../scss/mixins.scss";
+
+$node-width: 194px;
```

Comment:
@tealefristoe I believe it's possible to share variables between CSS and/or SASS and JavaScript. I agree it would be ideal to have things defined in one place and reused in the other when possible. I actually started to look into that, but haven't tried it before and was wary of how much work it might take to use here.

When I start working on the separate Color Picker story, I'll have a go at that if there's time. Maybe it'll turn out to be simple.

Reply:
Kirk actually brought this up in CLUE story hour, and there are examples of it in CLUE along with a link to a blog post that explains how to do it. I might try to incorporate it into the story I'm working on, but you shouldn't worry about it for this one.